var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -165,6 +167,13 @@ class Trilinos(CMakePackage):
     conflicts('+fortrilinos', when='~fortran')
     conflicts('+fortrilinos', when='@:99')
     conflicts('+fortrilinos', when='@master')
+    # Can only use one type of SuperLU
+    conflicts('+superlu-dist', when='+superlu')
+    # For Trilinos v11 we need to force SuperLUDist=OFF, since only the
+    # deprecated SuperLUDist v3.3 together with an Amesos patch is working.
+    conflicts('+superlu-dist', when='@:11.14.3')
```

Comment:
Not sure what their release methods look like, but if they ever add version 11.14.4 with bug fixes, you might want to change this to `when='@:11'`.